UI/data/themes/Yami.qss

```diff
@@ -135,7 +135,6 @@ SourceTree QWidget {
     border: 1px solid palette(dark);
 }
 
-
```

Comment:
This extra line appears in every theme file that used the same base, and it is not the only double blank line before a comment header/label. Might as well leave it be.

Reply:
I've force-pushed the change for this nit.